ggml.c

```diff
@@ -10810,6 +10815,28 @@ static void ggml_compute_forward_mul_mat(
     }
 #endif
 
+#if GGML_USE_LLAMAFILE
+    if (nb10 == ggml_type_size(src1->type)) {
+        for (int64_t j = 0; j < ne13; j++)
+            for (int64_t i = 0; i < ne12; i++)
```

Comment:
The condition should be sufficient.

Instead of `i` and `j` use `i12` and `i13`

Reply:
Done